site/docs/contribute/openapi.md

```diff
@@ -0,0 +1,13 @@
+# Using OpenAPI Generator to Generate Code for a Data Catalog Connector (in the Go Language)
+
+The Fybrik repository contains specification files that detail the data catalog connector API. These files include [datacatalog.spec.yaml](https://github.com/fybrik/fybrik/blob/master/connectors/api/datacatalog.spec.yaml) and [taxonomy.json](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/taxonomy/). They detail all the fields that the data catalog connector should expect for each of the supported operations: createAsset, getAssetInfo, deleteAsset, and updateAsset.
+
```

Comment:
@cdoron @rohithdv maybe add the command line to generate the code with openapi generator as it is currently not documented anywhere.